imagetool/src/main/java/com/oracle/weblogic/imagetool/installer/InstallerType.java

```diff
@@ -9,6 +9,7 @@
  */
 public enum InstallerType {
 
+    WLSLITE("wlslite"),
```

Comment:
WLSLITE can be dev, slim, generic how do you differentiate these types?

Reply:
good point, I will remove it.